pkg/products/rhssouser/reconciler.go

```diff
@@ -406,11 +767,11 @@ func (r *Reconciler) setupOpenshiftIDP(ctx context.Context, installation *integr
 
 	oauthc := &oauthv1.OAuthClient{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: r.getOAuthClientName(),
+			Name: r.getOAuthClientName(realm),
 		},
 		Secret: clientSecret,
 		RedirectURIs: []string{
-			r.Config.GetHost() + "/auth/realms/user-sso/broker/openshift-v4/endpoint",
+			host + "/auth/realms/" + realm + "/broker/openshift-v4/endpoint",
```

Comment:
Same as my other comment about the host.
Won't this use a route with wrong certificate?

Reply:
Yes, temporarily use self signed certs until  https://issues.redhat.com/browse/INTLY-5093 implemented

Reply:
That change was just merged to master btw :)

Reply:
I'm now creating the edge route before Realms in order to pass edge host URL to Realm.